src/library_int53.js

```diff
@@ -140,5 +153,9 @@ mergeInto(LibraryManager.library, {
 #if WASM_BIGINT
 global.i53ConversionDeps = ['$bigintToI53Checked'];
 #else
-global.i53ConversionDeps = ['$convertI32PairToI53Checked'];
+global.i53ConversionDeps = [
+  '$convertI32PairToI53Checked',
+  '$sendI53BitsLow',
+  '$sendI53BitsHigh',
```

Comment:
I'm not sure about this since the latter two will very rarely be used.. unlike the first one which is basically used everywhere.